package com.thinkinginjava.error_handling_with_exceptions.exercise.chapter12_10;

/**
 * page 271
 *   -- chapter_12 通过异常处理错误
 *
 *   12.10 构造器
 */